File: test_data/code/java/1748-sum-of-unique-elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
        Map<Integer, Integer> countMap = new HashMap<>();
        for (int num : nums) {
            countMap.put(num, countMap.getOrDefault(num, 0) + 1);
        }
        
        int sum = 0;
        for (Map.Entry<Integer, Integer> entry : countMap.entrySet()) {
            if (entry.getValue() == 1) {
                sum += entry.getKey();
            }
        }
        
        return sum;
    }
}

File: test_data/code/python/1748-sum-of-unique-elements.py
Python
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        num_count = {}
        for num in nums:
            if num in num_count:
                num_count[num] += 1
            else:
                num_count[num] = 1

        sum = 0
        for num, count in num_count.items():
            if count == 1:
                sum += num

        return sum